### ISSUE TITLE
Error when loading models with oobabot extension enabled

### ISSUE THREAD

**Issue by degreeze** [chrisrude/oobabot-plugin#26]
I'm getting error when trying to load different GPTQ models with **oobabot** extension enabled:
```
2023-07-28 23:21:31 INFO:Loading airoboros-33B-gpt4-1.3-GPTQ...
2023-07-28 23:21:36 ERROR:Failed to load the model.
Traceback (most recent call last):
  File "\textgen-webui\server.py", line 68, in load_model_wrapper
    shared.model, shared.tokenizer = load_model(shared.model_name, loader)
  File "\textgen-webui\modules\models.py", line 86, in load_model
    tokenizer = load_tokenizer(model_name, model)
  File "\textgen-webui\modules\models.py", line 103, in load_tokenizer
    tokenizer = AutoTokenizer.from_pretrained(
  File "\transformers\models\auto\tokenization_auto.py", line 702, in from_pretrained
    return tokenizer_class.from_pretrained(pretrained_model_name_or_path, *inputs, **kwargs)
  File "\transformers\tokenization_utils_base.py", line 1841, in from_pretrained
    return cls._from_pretrained(
  File "\transformers\tokenization_utils_base.py", line 2004, in _from_pretrained
    tokenizer = cls(*init_inputs, **init_kwargs)
  File "\transformers\models\llama\tokenization_llama.py", line 135, in __init__
    logger.warning_once(
  File "\transformers\utils\logging.py", line 295, in warning_once
    self.warning(*args, **kwargs)
  File "\envs\textgen\lib\logging\__init__.py", line 1489, in warning
    self._log(WARNING, msg, args, **kwargs)
  File "\envs\textgen\lib\logging\__init__.py", line 1624, in _log
    self.handle(record)
  File "\envs\textgen\lib\logging\__init__.py", line 1634, in handle
    self.callHandlers(record)
  File "\envs\textgen\lib\logging\__init__.py", line 1704, in callHandlers
    lastResort.handle(record)
  File "\envs\textgen\lib\logging\__init__.py", line 968, in handle
    self.emit(record)
  File "\textgen-webui\modules\logging_colors.py", line 74, in new
    args[0]._set_color(color)
AttributeError: '_StderrHandler' object has no attribute '_set_color'
```
When launching **oobabooga_text_webui** without this extension - every model I've tested loading just fine.
I'm on a latest commits for both **textgen_webui** ([d6314fd](https://github.com/oobabooga/text-generation-webui/commit/d6314fd5394bbfe2edd9030d953892dcfc4de105)) and **oobabot**. Actually, this is a fresh install from a clean conda env, just to remove potential conflicts.

Any ideas how to troubleshoot it further?

**Comment by chrisrude** [comment]
Thanks for reporting this!  This is definitely my fault, apologies.

I believe I've put in a fix for this in this commit, which should be fixed in version 0.2.2
https://github.com/chrisrude/oobabot-plugin/commit/84017e717b08bb2c75a112c70a913dfa7dcfa9b5

Could you please confirm if upgrading to 0.2.2 fixes it?  To do that, from the directory where you run `python server.py`, run:

`pip install oobabot-plugin --upgrade`

Again, apologies for the bug and let me know how if the fix works!

**Comment by degreeze** [comment]
Yes, upgrading to 0.2.2 fixed the model loading! Thanks for a quick response!
